File: 백준/Bronze/1546. 평균/평균.py
N = int(input())

scores = list(map(int, input().split()))

m = 0  # 최대 점수 찾기
for score in scores:
    if score > m:
        m = score

sum_new = 0
for s in scores:
    sum_new += (s / m * 100)

print(sum_new / N)

File: 백준/Bronze/1546. 평균/평균.js
function solution(n, list) {
  const max = Math.max(...list);
  let sum = 0;
  list.map((el) => {
    el = (el / max) * 100;
    sum += el;
  });
  console.log(sum / n);
}

const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];
rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  let n = parseInt(input[0]);
  let list = input[1].split(" ").map((el) => parseInt(el));
  solution(n, list);
  process.exit();
});
